sys/dev/ath/if_ath_usb_devlist.h

```diff
@@ -20,41 +20,7 @@
  */
 
```

Comment:
It's not part of this commit but I see this file has no protection for multiple/recursive inclusions: `#ifndef __IF_ATH_USB_DEVLIST_H__`

Reply:
This files was copy from original sys/dev/ath/if_ath_usb_devlist.h doesn't have such protection. I was wondering if I should fix it, but finally it was done in the same way as original file.

Reply:
Fixed